config/routes.rb

```diff
@@ -4,5 +4,6 @@
                      path_names: { sign_in: 'login', sign_out: 'logout' }
   root to: 'users#index'
   resources :users, only: %i[edit update show index]
+  get 'change_achieved', action: :change_achieved, controller: 'targets'
```

Comment:
You changed object.
If you want to retrieve object, use method `get`.
If you want to create object, use method `post`.
If you want to update object, use method `put`.